docs/advanced/adapt.rst

```diff
@@ -22,7 +22,7 @@ returned.
   or `~psycopg.Cursor`.
 
 - Every context object derived from another context inherits its adapters
-  mapping: cursors created from a connection inherit the connection's
+  mapping: cursors created from a connection inherit the connections
```

Comment:
maybre "the connection configuration"?

Reply:
I think this was clearer as it was. `connection's` with a possessive apostrophe indicating that we're talking about the configuration belonging to the connection (being inherited).

Reply:
OK, I'll revert that.

Reply:
Thank you for double checking, @asqui! I struggle a bit with the possessive case and unanimated objects, so I assumed I was wrong :)